### PR TITLE
Update Readme about note @vue/composition-api

### PR DESCRIPTION
**Pull request by geextudio**
@vue/composition-api is a plug-in of Vue 2.x which is expected to go with Vue in dependencies instead of devDependencies in package.json

### CODE REVIEW

**Comment by geextudio** [comment]
@vue/composition-api is a plug-in of Vue 2.x which is expected to go with Vue in dependencies instead of devDependencies in package.json

File: README.md [hunk 1]
```diff
@@ -23,7 +23,13 @@ $ npm install echarts vue-echarts
...
 
 ```sh
-npm i -D @vue/composition-api
+npm i @vue/composition-api
+```
+
+```javascript
+import VueCompositionAPI from '@vue/composition-api'
+
+Vue.use(VueCompositionAPI)
```
Comment (Justineo) [inline README.md:32]: You don't need to manually install the plugin as `vue-demi` already does that for you.

Comment (geextudio) [reply README.md:32]: ok. I think we may use npm i @vue/composition-api instead of npm i -D @vue/composition-api. ?

**Comment by geextudio** [comment]
it uses vue-demi

**Pull request closed without merging**